controllers/index.js

```diff
@@ -1,4 +1,12 @@
-const {retrieveTopics} = require('./models')
+const {retrieveEndpoints, retrieveTopics} = require('./models')
+
+exports.getEndpoints = (req, res, next) => {
+    return retrieveEndpoints()
+    .then((body) => {
+        res.status(200).send(body)
```

Comment:
We should send back data to the client on a meaningful key e.g, they should be able to access it on body.endpoints :)